.gitignore

```diff
@@ -26,6 +26,7 @@ win32ver.rc
 *.exe
 lib*dll.def
 lib*.pc
+.ccls-cache
```

Comment:
This looks like it's needed for your local tooling rather than the project's tooling. Please use your own local git setup files for this, I don't think this ignore rule belongs to the project's files (see $GIT_DIR/info/exclude).